src/sidebar/panel.js

```diff
@@ -26,14 +33,14 @@ function handleLocalContent(data) {
   if (!data.hasOwnProperty('notes')) {
     quill.setContents({
       ops: [
-        { attributes: { size: 'large', bold: true }, insert: 'Welcome!' },
-        { insert: '\n\n' },
+        { attributes: { size: 'large', bold: true }, insert: browser.i18n.getMessage('welcomeTitle') },
+        { insert: '\n\n', attributes: { direction: LANG_DIR, align: TEXT_ALIGN_DIR }},
```

Comment:
This is not going to work, it might for the initial text but then if you type text, you need somehow to tell quill.js about the text direction too.